package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.2",
   "description": "Scripts used to generate a font catalog for harp.gl",
   "scripts": {
+    "updateRanges": "ts-node ./scripts/unicode-ranges-generator.ts",
```

Comment:
Why not call update-ranges-generator as part of the build. This would avoid having the unicode-ranges.json in the repository.

Reply:
We should add it to the test step too, so the json file is there when required.